osc/commandline.py

```diff
@@ -2723,42 +2701,29 @@ def do_createrequest(self, subcmd, opts, *args):
 
         apiurl = self.get_api_url()
 
-        i = 0
         actionsxml = ""
         supersede = set()
-        for ai in opts.actions:
-            if ai == 'submit':
-                args = opts.actiondata[i]
-                i = i + 1
+        for actiondata in opts.actions:
+            action = actiondata[0]
+            args = actiondata[1:]
+            if action == 'submit':
                 actions, to_supersede = self._submit_request(args, opts, options_block)
                 actionsxml += actions
                 supersede.update(to_supersede)
-            elif ai == 'delete':
-                args = opts.actiondata[i]
+            elif action == 'delete':
                 actionsxml += self._delete_request(args, opts)
-                i = i + 1
-            elif ai == 'change_devel':
-                args = opts.actiondata[i]
+            elif action == 'change_devel':
                 actionsxml += self._changedevel_request(args, opts)
-                i = i + 1
-            elif ai == 'add_me':
-                args = opts.actiondata[i]
+            elif action == 'add_me':
                 actionsxml += self._add_me(args, opts)
-                i = i + 1
-            elif ai == 'add_group':
-                args = opts.actiondata[i]
+            elif action == 'add_group':
                 actionsxml += self._add_group(args, opts)
-                i = i + 1
-            elif ai == 'add_role':
-                args = opts.actiondata[i]
+            elif action == 'add_role':
                 actionsxml += self._add_user(args, opts)
-                i = i + 1
-            elif ai == 'set_bugowner':
-                args = opts.actiondata[i]
+            elif action == 'set_bugowner':
                 actionsxml += self._set_bugowner(args, opts)
-                i = i + 1
             else:
-                raise oscerr.WrongArgs('Unsupported action %s' % ai)
+                raise oscerr.WrongArgs('Unsupported action %s' % action)
```

Comment:
Would be nice to replace it with a double-quoted f-string:
`raise oscerr.WrongArgs(f"Unsupported action {action}")`